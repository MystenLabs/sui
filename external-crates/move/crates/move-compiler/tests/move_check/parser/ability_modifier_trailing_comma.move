address 0x42 {
module M {
    // invalid trailing comma
    struct Foo has copy, drop, {}
}
}
