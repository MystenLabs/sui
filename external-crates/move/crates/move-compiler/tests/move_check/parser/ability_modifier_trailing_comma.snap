error[E01002]: unexpected token
  ┌─ tests/move_check/parser/ability_modifier_trailing_comma.move:4:32
  │
4 │     struct Foo has copy, drop, {}
  │                                ^ Unexpected '{'. Expected a type ability, one of: 'copy', 'drop', 'store', or 'key'

